spec/locales-currencies-tz.html

```diff
@@ -205,41 +205,41 @@ <h1>DefaultTimeZone ( )</h1>
       </p>
 
       <emu-alg>
-        1. If the result of IsSanctionedSimpleUnitIdentifier(_unitIdentifier_) is *true*, then
+        1. If ! IsSanctionedSimpleUnitIdentifier(_unitIdentifier_) is *true*, then
           1. Return *true*.
-        1. If the substring *"-per-"* does not occur exactly once in _unitIdentifier_, then
+        1. Let _i_ be ! StringIndexOf(_unitIdentifier_, *"-per-"*, 0).
+        1. If _i_ is -1 or ! StringIndexOf(_unitIdentifier_, *"-per-"*, _i_ + 1) is not -1, then
           1. Return *false*.
-        1. Let _numerator_ be the substring of _unitIdentifier_ from the beginning to just before *"-per-"*.
-        1. If the result of IsSanctionedSimpleUnitIdentifier(_numerator_) is *false*, then
-          1. Return *false*.
-        1. Let _denominator_ be the substring of _unitIdentifier_ from just after *"-per-"* to the end.
-        1. If the result of IsSanctionedSimpleUnitIdentifier(_denominator_) is *false*, then
-          1. Return *false*.
-        1. Return *true*.
+        1. Assert: The five-character substring *"-per-"* occurs exactly once in _unitIdentifier_, at index _i_.
+        1. Let _numerator_ be the substring of _unitIdentifier_ from 0 to _i_.
+        1. Let _denominator_ be the substring of _unitIdentifier_ from _i_ + 5.
+        1. If ! IsSanctionedSimpleUnitIdentifier(_numerator_) and ! IsSanctionedSimpleUnitIdentifier(_denominator_) are both *true*, then
+          1. Return *true*.
+        1. Return *false*.
       </emu-alg>
     </emu-clause>
 
     <emu-clause id="sec-issanctionedsimpleunitidentifier" aoid="IsSanctionedSimpleUnitIdentifier">
       <h1>IsSanctionedSimpleUnitIdentifier ( _unitIdentifier_ )</h1>
 
       <p>
-        The IsSanctionedSimpleUnitIdentifier abstract operation verifies that the given core unit identifier is among the simple units sanctioned in the current version of the ECMAScript Internationalization API Specification, a subset of the Validity Data as described in <a href="https://unicode.org/reports/tr35/tr35.html#Validity_Data">UTS #35, Part 1, Section 3.11</a>; the list may grow over time. As discussed in UTS #35, a simple unit is one that does not have a numerator and denominator. The following steps are taken:
+        The IsSanctionedSimpleUnitIdentifier abstract operation verifies that the _unitIdentifier_ argument (which must be a String value) is among the single unit identifiers sanctioned in the current version of the ECMAScript Internationalization API Specification, which are a subset of the Common Locale Data Repository <a href="https://github.com/unicode-org/cldr/blob/maint/maint-38/common/validity/unit.xml">release 38 unit validity data</a>; the list may grow over time. As discussed in UTS #35, a single unit identifier is a core unit identifier that is not composed of multiplication or division of other unit identifiers. The following steps are taken:
```

Comment:
This text contains the only use of "current version of the ECMAScript Internationalization API Specification" that I could find (everywhere else uses "this specification" or "ECMAScript 2022 Internationalization API Specification"). A followup should reconcile that, ideally matching ECMA-262 (which seems to prefer "this specification").